README.rst

```diff
@@ -86,3 +86,16 @@ documents using absolute or relative local filesytem paths. If a relative path
 is referenced and that document isn't found in the current directory then the
 following locations will be searched:
 http://www.commonwl.org/v1.0/CommandLineTool.html#Discovering_CWL_documents_on_a_local_filesystem
+
+Use with Dockstore
```

Comment:
All references to Dockstore need to be replaced with "the GA4GH Tool Registry API"

At some point there should be a link to https://github.com/ga4gh/tool-registry-schemas#what-is-the-tool-registry-api-schema

Dockstore.org should still be mentioned as it is the default backend